linked_list/linked_list.py

```diff
@@ -0,0 +1,246 @@
+from __future__ import print_function  # For Python 2
+
+
+class Node(object):
+    def __init__(self, data=None, next_=None):
+        # Initialises Node
+        self.data = data
+        self.next = next_
+
+    def __str__(self):
+        return str(self.data)
+
+
+class Linked_list(object):
+    def __init__(self):
+        # Initialises Linked List.
+        self.head = None
+        self.size = 0
+
+    def __str__(self):
```

Comment:
this method should return the list, rather than print. Also it can be made shorter. 😉